daft/execution/physical_plan_factory.py

```diff
@@ -59,6 +59,9 @@ def get_physical_plan(node: LogicalPlan) -> Iterator[None | ExecutionStep[Partit
                 child_plan=child_plan,
                 pipeable_instruction=execution_step.Aggregate(to_agg=node._agg, group_by=node._group_by),
             )
+        
+        elif isinstance(node, logical_plan.LocalCount):
+            return physical_plan.local_count(child_plan, node)
```

Comment:
I believe local count is actually a pipelineable instruction. Once we do the refactors to use `vPartition.count_rows()`, we should be able to do:

```
return physical_plan.pipeline_instruction(
    child_plan=child_plan,
    pipeable_instruction=execution_step.LocalCount(),
)
```